src/ds/ds-private.h

```diff
@@ -35,6 +35,9 @@ namespace librealsense
         const uint8_t DS5_ENABLE_AUTO_EXPOSURE = 0xB;
         const uint8_t DS5_LED_PWR = 0xE;
         const uint8_t DS5_THERMAL_COMPENSATION = 0xF;
+        
+        //DS6 safety XU identifiers
+        const uint16_t DS6_ACTIVE_SAFETY_PRESET = 0x02;
```

Comment:
Please move to ds6-private under `xu_id` namespace (like in my PR)
This is not a common ID

Reply:
done